src/components/generic/InputNumberNoScrollWithUnit/InputNumberNoScrollWithUnit.js

```diff
@@ -28,11 +28,40 @@ const InnerInput = styled(InputNumberNoScroll)`
   padding: ${theme.spacing.xsmall};
 `
 
-const InputNumberNoScrollWithUnit = ({ unit, alignUnitsLeft = false, ...restOfProps }) => {
+const InputNumberNoScrollWithUnit = ({
+  unit = '',
+  alignUnitsLeft = false,
+  value = '',
+  onChange = () => {},
+  name = '',
+  ...restOfProps
+}) => {
+  const [inputValue, setInputValue] = useState(value || '')
+
+  const handleInputChange = (e) => {
```

Comment:
nitpick: `event` instead of `e`

Reply:
Do we want to handle this outside of the generic component, so use the handler only if its defined in props? As is, this will make _all_ `InputNumberNoScrollWithUnit` be restricted to 2 decimal places.

Reply:
yeah I was thinking about that - will update